client/package.json

```diff
@@ -24,11 +28,15 @@
     "axios": "^1.7.2",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
+    "color": "^4.2.3",
+    "deck.gl": "^9.0.17",
     "express": "^4.19.2",
+    "jotai": "^2.8.2",
     "lucide-react": "^0.378.0",
     "mapbox-gl": "^3.3.0",
     "next": "14.2.3",
     "next-intl": "^3.14.1",
+    "next-usequerystate": "^1.17.4",
```

Comment:
use `nuqs` instead.

https://www.npmjs.com/package/next-usequerystate check the table, for next versions higher than 14.0.4 we should use nuqs